settings.gradle

```diff
@@ -41,4 +42,4 @@ include 'fabric-rendering-fluids-v1'
 include 'fabric-resource-loader-v0'
 include 'fabric-tag-extensions-v0'
 include 'fabric-textures-v0'
-include 'fabric-dimensions-v1'
\ No newline at end of file
+include 'fabric-dimensions-v1'
```

Comment:
TDLR: This change is perfectly fine since the old one had an extra space on the end of it, this change makes it conform to the rest of the entries.

Reply:
Not quite - the file didn't end with a newline (which all text files should). Still a perfectly fine change.

Reply:
Maybe fix this with merge conflicts.